core/client/lemverse.js

```diff
@@ -455,6 +455,7 @@ Template.lemverse.onDestroyed(function () {
 Template.lemverse.helpers({
   allRemoteStreamsByUsers: () => peer.remoteStreamsByUsers.get(),
   guest: () => Meteor.user({ fields: { 'profile.guest': 1 } })?.profile.guest,
+  onboarding: () => Meteor.user()?.profile.guest && !Meteor.user()?.username,
```

Comment:
In a helper it is better to specify the fields you are going to use (as on the line above). Otherwise Meteor will go through this code every time the user moves, which can be quite heavy 😅

Reply:
Oh okay, didn't know that